test/features/pagecontent/pagecontent.js

```diff
@@ -28,6 +29,15 @@ describe('item requests', function() {
     function deniedContentUri(format) {
         return [server.config.bucketURL(), format, encodeURIComponent(deniedTitle), deniedRev].join('/');
     }
+
+    const sysURL = (domain = server.config.defaultDomain) => `${hostPort}/${domain}/sys_passthrough`;
+
+    const sysGet = (path, opt = {}) => {
+        const domain = opt.domain || Server.DEFAULT_DOMAIN;
+        const uri = `${sysURL(domain)}/${path}`;
+        return preq.get( { uri, ...opt } );
```

Comment:
Here you are passing `domain` as an attribute to the underlying request library and it throws an error that is not handled that's why it gets stuck.